rs/nervous_system/agent/src/pocketic_impl.rs

```diff
@@ -160,6 +161,15 @@ impl CallCanisters for PocketIcAgent<'_> {
     fn caller(&self) -> Result<Principal, Self::Error> {
         Ok(self.sender)
     }
+
+    fn is_canister_stopped_error(&self, err: &Self::Error) -> bool {
+        match err {
+            PocketIcCallError::PocketIc(err) => {
```

Comment:
This could work as well.
```
PocketIcCallError::PocketIc(ErrorCode::CanisterStopped |  ErrorCode::CanisterStopping) => true
```

Reply:
`ErrorCode::CanisterStopped` is the value of `error_code` field from `err`, so I'm not sure if this is going to work